addons/project_purchase/tests/test_project_profitability.py

```diff
@@ -754,3 +754,39 @@ def test_vendor_credit_note_profitability(self):
             'billed': 0.0,
             'to_bill': 0.0,
         })
+
+    def test_expense_from_PO_is_in_profitability(self):
+        product = self.env['product.product'].create([{
+            'name': 'Test Product',
+            'standard_price': 30,
+            'list_price': 90,
+            'type': 'service',
+            'service_tracking': 'task_in_project',
+            'expense_policy': 'sales_price',
```

Comment:
It's inaccessible if the product_ok is False so please add purchase_ok (don't think it's the default behaviour but let's stay UX-accurate)